src/picknik_ur_site_config/objectives/open_lever_handle_door.xml

```diff
@@ -5,6 +5,10 @@
         <Control ID="Sequence" name="Setup">
             <Action ID="LoadObjectiveParameters"
                  config_file_name="open_lever_handle_door_config.yaml" parameters="{parameters}" />
+            <Decorator ID="ForceSuccess">
```

Comment:
Is this so the Objective still runs in Gazebo where there is no admittance controller? If so, might be worth dropping a comment.

Reply:
Yeah this is why. It also has some of the odd behavior (same as the Tare FTS service) where if planning fails then it also complains it can't find the correct controller. In the future we should try to fix that for both of these. In the meantime I'll just update the comments.

Reply:
Updated comments. I think it should be included so it works on sim (even if the behavior otherwise fails for different reasons such as our scene not having what it needs). Issue so we can fix this properly.